Tests/MongoMobileTests/MongoMobileTests.swift

```diff
@@ -1,19 +1,76 @@
-@testable import MongoMobile
 import XCTest
+import MongoSwift
+
+@testable import MongoMobile
 
 final class MongoMobileTests: XCTestCase {
-    static var allTests: [(String, (ClientTests) -> () throws -> Void)] {
+    static var allTests: [(String, (MongoMobileTests) -> () throws -> Void)] {
         return [
-            ("testMongoMobile", testMongoMobile)
+            ("testMongoMobileBasic", testMongoMobileBasic),
+            ("testSequentialAccess", testSequentialAccess)
         ]
     }
 
-    func testMongoMobile() throws {
-        let client = MongoMobile.create(settings: MongoClientSettings(dbPath: "test-path"))
+    // NOTE: These only works because we have one test suite. These method are called
```

Comment:
Yeah, we looked for something previously and I think the only approach that always works is to define a [PrincipalClass](https://developer.apple.com/documentation/xctest/xctestobservationcenter) for the test. Unfortunately, this only works on OSX :/  Since we are unlikely to have multiple test suites for this project, I think this is probably fine for now.

Reply:
I think that doesn't even work for one-time teardown sadly, see my comment on [SWIFT-118](https://jira.mongodb.org/browse/SWIFT-118)